grijanje.py

```diff
@@ -0,0 +1,20 @@
+class Temperatura:
+    def __init__(self, tempereatura=22):  
+        self.temperatura = 22
```

Comment:
Zašto imamo argument (u kojem btw imaš tipfeler) ako `self.temperatura` uvijek postavimo na 22?